app/src/test/java/ru/vssemikoz/newsfeed/usacases/GetFilteredNewsUseCaseTest.java

```diff
@@ -73,14 +74,14 @@ public void verifyGetFilteredIsCalled() {
 
     @Test
     public void verifyGetFilteredReturnNewsList() {
-        when(newsStorage.getFiltered(any(), any())).thenReturn(exampleNewsList);
-        assertEquals(getFilteredNewsUseCase.run(paramsExample), exampleNewsList);
+        when(newsStorage.getFiltered(any(), any())).thenReturn(Single.just(exampleNewsList));
+        getFilteredNewsUseCase.run(paramsExample).test().assertValue(exampleNewsList);
```

Comment:
Looks cool! But, you're testing a chain and should format it like a chain. BTW, where's `assertNoErrors()`?